.github/workflows/machine-os-pr.yml

```diff
@@ -0,0 +1,145 @@
+          else
+            echo "update=true" >> "$GITHUB_OUTPUT"
+          fi
+          echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+
+      - name: Check machine-os-branch
+        if: steps.getversion.outputs.update == 'true'
+        run: |
+          if ! (curl -s https://api.github.com/repos/$UPSTREAM_MACHINE_OS/branches| jq -e --arg branch "${{github.base_ref}}" '.[] | select(.name==$branch)') ; then
+            echo "::error:: Release branch does not exist."
+            echo "::error:: Please push $branch to $UPSTREAM_MACHINE_OS, then re-run this task."
+            exit 1
+          fi
+
+      - name: Label
+        if: steps.getversion.outputs.update == 'true'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+        run: |
+          gh pr edit --add-label do-not-merge/wait-machine-os-build ${{github.event.number}}
+
+      - name: Install wait-for-copr
+        if: steps.getversion.outputs.update == 'true'
+        run: |
+          pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+      - uses: actions/checkout@v4
+        if: steps.getversion.outputs.update == 'true'
+        id: checkout
+        with:
+          repository: containers/podman-machine-os
+          ref: ${{github.base_ref}}
+          token: ${{secrets.PODMANBOT_TOKEN}}
+
+      - name: Bump version
+        if: steps.getversion.outputs.update == 'true'
+        run: |
+          echo ${{steps.getversion.outputs.version}}
+          sed -i 's/export PODMAN_VERSION=".*"/export PODMAN_VERSION="${{steps.getversion.outputs.version}}"/g' podman-rpm-info-vars.sh
+          sed -i 's/export PODMAN_PR_NUM=".*"/export PODMAN_PR_NUM="${{github.event.number}}"/g' podman-rpm-info-vars.sh
+          echo "Updated file:"
+          cat podman-rpm-info-vars.sh
+
+      - name: Wait for COPR build
+        if: steps.getversion.outputs.update == 'true'
+        run: |
+          wait-for-copr \
+          --owner packit \
+          --project containers-podman-${{github.event.number}} \
+          podman \
+          ${SHA::9}
+
+      - name: Push
+        if: steps.getversion.outputs.update == 'true'
+        run: |
+          # Make committer the user who triggered the action, either through cutting a release or manual trigger
+          # GitHub gives everyone a noreply email associated with their account, use that email for the sign-off
+          git config --local user.name ${{ github.actor }}
+          git config --local user.email "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
+          bumpbranch="pr${{github.event.number}}"
+          git checkout -b $bumpbranch
+          git add podman-rpm-info-vars.sh
+          git commit --signoff -m "Bump Podman to v${{ steps.getversion.outputs.version }}"
+          git remote add podmanbot https://github.com/podmanbot/podman-machine-os
+          git push -f podmanbot "$bumpbranch"
```

Comment:
Do we need `-f` here?

Reply:
If the release pr is re-pushed for some reason, the machine-os-build pr is also re-pushed the so the build runs again with the COPR build.

Reply:
sgtm